src/app/store/reducers/login.reducers.ts

```diff
@@ -0,0 +1,22 @@
+import {createReducer, on} from "@ngrx/store";
+import {initialLoginState} from "../state/login.state";
+import {loginUser, loginUserFailure, loginUserSuccess} from "../actions/login.actions";
+
+
+export const loginReducers = createReducer(
+  initialLoginState,
+  on(loginUser, (state) => ({
+    ...state,
+    status: 'loading'
+  })),
+  on(loginUserSuccess, (state, { user }) => ({
+    ...state,
+    loggedUser: user,
+    status: 'success'
+  })),
+  on(loginUserFailure, (state, { error }) => ({
+    ...state,
+    error: error,
```

Comment:
Imo, error shouldn't be within the state. It should be passed only as argument to the failure action and shown there.